shared/common-adapters/list-item.native.js

```diff
@@ -31,6 +31,7 @@ export default class ListItem extends Component<void, Props, void> {
       <TouchableHighlight
         activeOpacity={0.8}
         underlayColor={globalColors.white}
+        action={<Box />}
```

Comment:
What does this line do? Couldn't find docs on an action prop for `TouchableHighlight`.